### PR TITLE
new(falco): add falco_libs.snaplen option, deprecate -S / --snaplen

### PR DESCRIPTION
**Pull request by LucaGuerra**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind cleanup

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area engine

**What this PR does / why we need it**:

As per title. This may be the last CLI option that we couldn't replicate in the falco config file.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
new(falco): add falco_libs.snaplen option, deprecate -S / --snaplen
```

### CODE REVIEW

File: userspace/falco/app/actions/init_inspectors.cpp [hunk 1]
```diff
@@ -87,8 +87,12 @@ static void init_syscall_inspector(falco::app::state& s, std::shared_ptr<sinsp>
...
 	//
-	// If required, set the snaplen
+	// If required, set the snaplen.
+	// In case both config and CLI options are specified, CLI takes precedence.
 	//
+	if(s.config->m_falco_libs_snaplen != 0) {
+		inspector->set_snaplen(s.config->m_falco_libs_snaplen);
+	}
 	if(s.options.snaplen != 0) {
 		inspector->set_snaplen(s.options.snaplen);
```
Comment (FedeDP) [inline userspace/falco/app/actions/init_inspectors.cpp:97]: Do we want to add the `attention this is deprectated and will be removed in..." log here? Like this one: https://github.com/falcosecurity/falco/pull/3329/files#diff-e1e4257c43745e29751e72ffa753fd44ec931b07b1bec5035f4d1b04d061848cR100

**Comment by LucaGuerra** [comment]
@FedeDP done!

**Review by FedeDP — APPROVED** [review]
/approve

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**